packages/dso-toolkit/src/components/description/description.stories-of.ts

```diff
@@ -31,15 +29,13 @@ interface DescriptionStoriesParameters<Implementation, Templates, TemplateFnRetu
 
 export interface DescriptionTemplates<TemplateFnReturnType> {
   descriptionTemplate: (descriptionProperties: Description) => TemplateFnReturnType;
-  exampleTemplate: (exampleData: ReturnType<typeof descriptionExample>) => TemplateFnReturnType;
+  exampleTemplate: (exampleData: (string | Description)[]) => TemplateFnReturnType;
 }
 
-export function descriptionMeta<TRenderer extends Renderer>({ readme }: MetaOptions = {}): ComponentAnnotations<
-  TRenderer,
-  DescriptionArgs | DescriptionExampleArgs
-> {
+export function descriptionMeta<TRenderer extends Renderer>({
```

Comment:
Wat er aan de default export wordt meegegeven dient als basis voor alle andere stories. Er is een gelaagdheid:

* Storybook globals
* Story Meta
* Story

Op alle niveau's kun je alles meegeven. Het meest specifieke wint. Maar hoe Description wordt geëtaleerd zit de gelaagdheid in de weg. Er zijn twee setjes aan args: `DescriptionExampleArgs` en `DescriptionArgs`. Deze zijn gelijkwaardig aan het elkaar, de een stapelt niet op de ander.

Dus op "Storybook global" en Story Meta niveau wordt er niets gezet. De twee Story's definieren hun eigen Args.

Omdat Meta geen args (en argTypes) meer geeft, is de typering op dit niveau niet meer nodig.

In plaats daarvan is het expliciet gemaakt welke Args de stories nodig hebben in de `interface DescriptionStories`.